conf/routes

```diff
@@ -5,6 +5,8 @@
 # An example controller showing a sample home page
   GET     /                           controllers.HomeController.index
   GET     /search/                    controllers.HomeController.query(query)
+  GET     /version                    controllers.SettingsController.version
+  GET     /stop                       controllers.SettingsController.stop
```

Comment:
This endpoint currently stops me from accepting this PR. 
We cannot do a `stop` endpoint without authentication of some sort. 
This is - as discussed - a no go.

Reply:
I removed the endpoint /stop for now. It will be re-introduced once we have developed our global authentication concept. Since we found out that docker containers actually do shut down gracefully (meaning sending ctrl+c) once you call ```docker stop``` the only use-case for this endpoint is to shut down an instance that was *not* deployed as a docker container.